Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -3524,6 +3555,12 @@ internal static void InitializeInEditor(IInputRuntime runtime = null)
             }
             s_SystemObject.newInputBackendsCheckedAsEnabled = true;
 
+#if UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS
+            // Make sure project wide input actions are enabled
+            if (actions != null)
```

Comment:
This is the second time actions are enabled in this code path since also done via Reset above. Needs to be checked which of these is correct since Reset is also invoked when static state stack is pushed/popped.

Reply:
The one in Reset is not used in Editor during the initialisation flow as its too early and at that point the value is null. So this line is needed then.